control-plane/plugin/src/resources/node.rs

```diff
@@ -220,18 +220,17 @@ impl GetWithArgs for Node {
     type Args = GetNodeArgs;
     async fn get(id: &Self::ID, args: &Self::Args, output: &utils::OutputFormat) -> PluginResult {
         match RestClient::client().nodes_api().get_node(id).await {
-            Ok(node) => {
-                let node_display =
-                    NodeDisplayLabels::new(node.clone().into_body(), args.show_labels());
-                match output {
-                    OutputFormat::Yaml | OutputFormat::Json => {
-                        print_table(output, node_display.inner);
-                    }
-                    OutputFormat::None => {
-                        print_table(output, node_display);
-                    }
+            Ok(node) => match output {
+                OutputFormat::Yaml | OutputFormat::Json => {
+                    print_table(output, node.clone().into_body());
                 }
-            }
+                OutputFormat::None => {
+                    print_table(
+                        output,
+                        NodeDisplayLabels::new(node.into_body(), args.show_labels()),
```

Comment:
if output format is None, We are still showing in array(`[]`) format ?

Reply:
yes, because we have honour `show-labels` in case on None. 
For json and YAML 
The spec will show the node labels and there is no need to have the `show-label` flag in that case